integration/fabric/atsa/chaincode/chaincode/asset_transfer.go

```diff
@@ -437,24 +436,24 @@ func transferAssetState(ctx contractapi.TransactionContextInterface, asset *Asse
 		return fmt.Errorf("failed to create composite key for receipt: %v", err)
 	}
 
-	txTimestamp, err := ctx.GetStub().GetTxTimestamp()
-	if err != nil {
-		return fmt.Errorf("failed to create timestamp for receipt: %v", err)
-	}
-
-	if err := txTimestamp.CheckValid(); err != nil {
-		return err
-	}
-	timestamp := txTimestamp.AsTime()
-
-	assetReceipt := receipt{
-		price:     price,
-		timestamp: timestamp,
-	}
-	receipt, err := json.Marshal(assetReceipt)
-	if err != nil {
-		return fmt.Errorf("failed to marshal receipt: %v", err)
-	}
+	//txTimestamp, err := ctx.GetStub().GetTxTimestamp()
```

Comment:
Removing this logic doesn't seem to break any tests and from a static code analysis, it seems redundant. Can I remove it?

Reply:
Are you sure that static check can catch this one? Where should be other other impl of this redundant code?

Reply:
this code is only written for the test, so if the integration test doesn't fail there I don't think there should be any other reason for it to be there. This code is not repeated somewhere else in the codebase. I didn't find a reason for it to be there from its first commit and I didn't find any file where `receipt` implemented `Marshaler` (or `Unmarshaler`)